fields.yml

```diff
@@ -1189,7 +1189,14 @@
           type: keyword
           example: "Mac OS X"
           description: >
-            Operating system name.
+            Operating system name, without the version.
+    
+        - name: full
+          level: extended
+          type: keyword
+          example: "Mac OS X 10.15.1"
+          description: >
+            Operating system name, including the version.
```

Comment:
I would make it more generic, so it could also contain for example `El Capitan` etc.

Reply:
Ah, like "including the version or code name"?

Reply:
done